SwanCustomEnvironments/swancustomenvironments/templates/customenvs.html

```diff
@@ -237,31 +237,33 @@ <h2 id="modal-title" style="text-align: left;"></h2>
                 // Decode the chunk into text and split by newlines
                 const chunked_lines = decoder.decode(value, { stream: true });
                 chunked_lines.split('\n').forEach(message_line => {
-                    if (message_line.startsWith('REPO_PATH')) {
-                        const repo_path = message_line.split(':')[1];
-                        if (repo_path && repo_path !== '/') {
-                            project_path = `/tree${repo_path}${file}`;
-                        }
-                    } 
-                    // Get the environment name from the output
-                    else if (message_line.startsWith('ENV_NAME')) {
-                        env_name = message_line.split(':')[1];
-                    } 
-                    // Environment already exists -> pops up a modal
-                    else if (message_line.startsWith('ENVIRONMENT_ALREADY_EXISTS')) {
-                        is_created = true;
-                        env_name = message_line.split(':')[1];
-                    } 
-                    // Usual case -> update output box with the message received
-                    else {
-                        outputBox.innerHTML += message_line;
-                        outputBox.scrollTop = outputBox.scrollHeight;
-                        if (message_line.includes('ERROR')) {
-                            has_failed = true;
+                    if (message_line !== '') {
```

Comment:
If a line just contains a line break, it arrives here as an empty string I guess? Aren't with this change omitting the empty lines, and therefore they won't appear in the output?